hack/schemas/main.go

```diff
@@ -31,6 +31,8 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/xeipuuv/gojsonschema"
+
```

Comment:
I think this new line on 35 is the cause of your `linter` error
https://app.travis-ci.com/github/GoogleContainerTools/skaffold/jobs/530055237#L316

Reply:
```
import (
  // stdlib

  // third-party

  // other packages of that project
)
```

https://stackoverflow.com/a/59964885/4540147

Reply:
Thanks!!!